sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -170,6 +171,11 @@ case class DropFunctionCommand(
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
     val catalog = sparkSession.sessionState.catalog
+
+    if (FunctionsCommand.virtualOperators.contains(functionName.toLowerCase(Locale.ROOT))) {
+      throw new AnalysisException(s"Cannot drop virtual function '$functionName'")
```

Comment:
ditto, we should make sure the behavior is consistent with other operators.

Reply:
> ditto

OK,  
`drop function `!=\case\between\<>``
here it will through exception of function not found